<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tests</class>
 <widget class="QWidget" name="Tests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QPushButton" name="more_4">
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/more.png</normaloff>:/icons/images/more.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="fourthAnsw">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>Answer4</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="fifthAnsw">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>Answer5</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="thirdAnsw">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: rgb(255, 93, 93);
	border: 4px solid rgb(170, 0, 0);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>Answer3</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="setReminderBut">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.841346, y2:0.756, stop:0 rgba(170, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 0, 255);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>Поставить напоминание</string>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/reminder.png</normaloff>:/icons/images/reminder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="firstAnsw">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>12345678901234567890123456789012345678</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="more_3">
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/more.png</normaloff>:/icons/images/more.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QPushButton" name="nextQuestBut">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.841346, y2:0.756, stop:0 rgba(170, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 0, 255);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>Следующий вопрос</string>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/arrowNext.png</normaloff>:/icons/images/arrowNext.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="more_2">
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/more.png</normaloff>:/icons/images/more.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
	text-align: center;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 85, 255, 255), stop:1 rgba(255, 255, 255, 255));
    width: 20px;
}</string>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
     <property name="format">
      <string>%v</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="more_5">
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/more.png</normaloff>:/icons/images/more.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="secondAnsw">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: rgb(0, 85, 0);
	background-color: rgb(85, 255, 127);
	border: 4px solid rgb(0, 170, 0);
	padding: 6px;
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string>Answer2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="more_1">
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color: black;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.884615, y2:0.892, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid rgb(0, 85, 255);
	border-radius: 20px 10px;
}

QPushButton:pressed{
	color: red;
	border: 4px solid red;
	border-radius: 20px 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="rsrc.qrc">
       <normaloff>:/icons/images/more.png</normaloff>:/icons/images/more.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextBrowser" name="questionText">
     <property name="styleSheet">
      <string notr="true">QTextBrowser{
	font: 9pt &quot;Times New Roman&quot;;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt;&quot;&gt;Question text&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
